src/components/FilterData.tsx

```diff
@@ -0,0 +1,111 @@
+import { AstarteInterfaceValues } from 'astarte-client';
```

Comment:
beside missing the license header, this component name is super generic.
`FilterData`, but which filter? which data? how is that done?

This component is infact very specific and only used in the datastream data visualization page, so i'll keep it's definition there for now